code/modules/surgery/bodyparts/dismemberment.dm

```diff
@@ -58,7 +58,8 @@
 	var/organ_spilled = 0
 	var/turf/T = get_turf(C)
 	C.add_splatter_floor(T)
-	playsound(get_turf(C), 'sound/misc/splort.ogg', 80, 1)
+	if(status != BODYPART_ROBOTIC)
+		playsound(get_turf(C), 'sound/misc/splort.ogg', 80, 1)
```

Comment:
Wish we had a better audio for mechanical limbs, but oh well.

Reply:
Could just the robot gib sound

Reply:
Would be better to get an original one; it's fine to leave this as is if you can't find anything pretty.